js/modules/loki_message_api.js

```diff
@@ -32,6 +32,41 @@ const calcNonce = (messageEventData, pubKey, data64, timestamp, ttl) => {
   return callWorker('calcPoW', timestamp, ttl, pubKey, data64, difficulty);
 };
 
+// we don't throw or catch here
+// mark private (_ prefix) since no error handling is done here...
+async function _retrieveNextMessages(nodeData, pubkey) {
+  const params = {
+    pubKey: pubkey,
+    lastHash: nodeData.lastHash || '',
+  };
+  const options = {
+    timeout: 40000,
+    ourPubKey: pubkey,
+  };
+
+  // let exceptions bubble up
+  const result = await lokiRpc(
+    `https://${nodeData.ip}`,
+    nodeData.port,
+    'retrieve',
+    params,
+    options,
+    '/storage_rpc/v1',
+    nodeData
+  );
+
+  if (result === false) {
+    // make a note of it because of caller doesn't care...
+    log.warn(
+      `loki_message:::_retrieveNextMessages - lokiRpc could not talk to ${
+        nodeData.ip
+      }:${nodeData.port}`
+    );
+  }
+
+  return result.messages || [];
```

Comment:
```suggestion
  return (result && result.messages) || [];
```
Need to do this because result can be false, which would cause the an exception on the line

Reply:
FIxed by returning `[]`.